vimdoc/module.py

```diff
@@ -60,11 +60,17 @@ def Merge(self, block, namespace=None):
       # Overwrite existing section if it's a default.
       if block_id not in self.sections or self.sections[block_id].IsDefault():
```

Comment:
I actually tried that at first, but there's a third case that's missing (default/default shouldn't throw error). Just played with a few variations on inverting the check again but didn't find any better version.